client/rpc/src/eth/block.rs

```diff
@@ -132,7 +132,22 @@ where
 					base_fee,
 				)))
 			}
-			_ => Ok(None),
+			_ => {
+				if let BlockNumber::Num(block_number) = number {
+					let eth_block = empty_block_from(block_number.into());
+					let eth_hash =
+						H256::from_slice(keccak_256(&rlp::encode(&eth_block.header)).as_slice());
+					Ok(Some(rich_block_build(
+						eth_block,
+						Default::default(),
+						Some(eth_hash),
+						full,
+						None,
```

Comment:
Should we put `base_fee` here?

Reply:
This is just empty block for no-evm period of Shiden, not sure it matters.